external/artinis/ft_nirs_prepare_ODtransformation.m

```diff
@@ -185,13 +185,21 @@
 coefs = cell2mat(textscan(fid, '%f %f %f %f %f'));
 
 % extract all optode combinations that are relevant here
+<<<<<<< HEAD
```

Comment:
one of the two alternatives is correct, and the <<<HEAD ===== lines need to be removed

Reply:
I just fixed it

Reply:
oh... I might not have fixed it. I just selected one at random. So this still needs to be looked at